### PR TITLE
contrib/hbssl: Support for OpenSSL 3

### PR DESCRIPTION
**Pull request by EMG60**
Support for Openssl 3

### CODE REVIEW

**Comment by tuffnatty** [comment]
@EMG60 Thanks! But now it looks like the line ending encoding in ChangeLog.txt has changed, that is, every line of the file is considered as modified. Please try to minimise unnecessary changes so that the remaining ones can be easily reviewed.

**Comment by EMG60** [comment]
Il 31/10/2023 11:18, Phil Krylov ha scritto:

> @EMG60 <https://github.com/EMG60> Thanks! But now it looks like the line 
> ending encoding in ChangeLog.txt has changed, that is, every line of the 
> file is considered as modified. Please try to minimise unnecessary 
> changes so that the remaining ones can be easily reviewed.

Fixed. Please check now.

-- 
Enrico Maria Giordano

http://www.emagsoftware.it
http://www.emagsoftware.it/emgmusic
http://www.emagsoftware.it/spectrum
http://www.emagsoftware.it/tbosg

**Comment by tuffnatty** [comment]
@EMG60 It's much better now! But UTF-8 characters are corrupted in ChangeLog.txt.

**Comment by EMG60** [comment]
Il 31/10/2023 12:21, Phil Krylov ha scritto:

> @EMG60 <https://github.com/EMG60> It's much better now! But UTF-8 
> characters are corrupted in ChangeLog.txt.

Recheck now and let me know.

-- 
Enrico Maria Giordano

http://www.emagsoftware.it
http://www.emagsoftware.it/emgmusic
http://www.emagsoftware.it/spectrum
http://www.emagsoftware.it/tbosg

**Comment by EMG60** [comment]
Still waiting... :-)

**Comment by tuffnatty** [comment]
@EMG60 Looks good now! I would merge it if I could test it. Unfortunately, I am missing a Windows test environment.

**Comment by EMG60** [comment]
So, what do I have to do now? I am near to give up. Is Harbour still an 
active project?

-- 
Enrico Maria Giordano

http://www.emagsoftware.it
http://www.emagsoftware.it/emgmusic
http://www.emagsoftware.it/spectrum
http://www.emagsoftware.it/tbosg

**Comment by EMG60** [comment]
Anyway, I already tested it building Harbour for MSC32, MSC64, BCC32, 
BCC64, MGW32 and MGW64. Is it not enough? Here they are the packages:

https://github.com/FiveTechSoft/harbour_and_xharbour_builds/blob/master/README.md

-- 
Enrico Maria Giordano

http://www.emagsoftware.it
http://www.emagsoftware.it/emgmusic
http://www.emagsoftware.it/spectrum
http://www.emagsoftware.it/tbosg

File: ChangeLog.txt [hunk 1]
```diff
@@ -7,6 +7,10 @@
    Entries may not always be in chronological/commit order.
    See license at the end of file. */
 
+2023-10-29 14:27 UTC+0100 Enrico Maria Giordano (e.m.giordano@emagsoftware.it)
+  * contrib\hbssl\hbssl,hbp
```
Comment (tuffnatty) [inline ChangeLog.txt:11]: please replace backslashes with forward slashes, and comma with dot.

**Review by tuffnatty — CHANGES_REQUESTED** [review]

**Comment by tuffnatty** [comment]
Thanks for your patience! If you have tested the Windows builds, I am going to merge this as soon as you fix the minor typos in Changelog.

**Comment by EMG60** [comment]
Ops! Sorry. Fixed.

**Comment by EMG60** [comment]
Thank you! :-)

-- 
Enrico Maria Giordano

http://www.emagsoftware.it
http://www.emagsoftware.it/emgmusic
http://www.emagsoftware.it/spectrum
http://www.emagsoftware.it/tbosg

**Pull request merged**